src/org-units-selector/OrgUnitsSelector.jsx

```diff
@@ -133,17 +176,8 @@ export default class OrgUnitsSelector extends React.Component {
     };
 
     filterOrgUnits = async value => {
-        const roots = await (!value
-            ? getDefaultRoots(this.props.d2)
-            : this.props.d2.models.organisationUnits
-                  .list({
-                      paging: true,
-                      pageSize: 10,
-                      fields: "id,displayName,path",
-                      filter: `displayName:ilike:${value}`,
-                  })
-                  .then(collection => collection.toArray()));
-
+        const opts = !value ? {} : { filter: `displayName:ilike:${value}` };
```

Comment:
If the signature is changed this could also look like ```const opts = value ? { filter: `displayName:ilike:${value}` } : null;```.

Reply:
You mean `undefined`?

Reply:
Yeah

Reply:
Done!